### PR TITLE
example improved

### PR DESCRIPTION
**Pull request by Chekushkin**

### CODE REVIEW

File: ruby/spec/saltedge_spec.rb [hunk 1]
```diff
@@ -43,13 +43,14 @@
 
   describe "#signature" do
     it "should return encrypted signature" do
-      item    = {
+      file = File.open("private.pem")
```
Comment (idanci) [inline ruby/spec/saltedge_spec.rb]: not used

File: ruby/private.pem [hunk 2]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (idanci) [inline ruby/private.pem]: Add note to readme that one should not use these keys for his app. He should generate his own pair.

Comment (deltadd) [inline ruby/private.pem]: just remove it from repository

File: ruby/public.pem [hunk 3]
```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```
Comment (deltadd) [inline ruby/public.pem]: also remove, use stubs

**Pull request merged**